src/components/List/List.js

```diff
@@ -10,13 +11,19 @@ class TypePad extends Component {
   }
 
   render() {
+    const { handleButtonClick, handleOpperate } = this.props;
     return (
       <ul className="flex">
-        <MainList />
-        <OperationList />
+        <MainList handleButtonClick={handleButtonClick} />
+        <OperationList handleButtonClick={handleButtonClick} handleOpperate={handleOpperate} />
```

Comment:
- [ ] According to the project requirements:

![image](https://user-images.githubusercontent.com/57421823/152986151-493ae25a-65af-425e-ae2b-cf65dad70443.png)

Therefore, kindly let's implement the use of React lifecycle methods (you can keep it simple), so we can move closer to approval 💪

Reply:
I have added a lifecycle method in Calculator.js.